src/components/ec-radio-btn/types.ts

```diff
@@ -0,0 +1,24 @@
+export enum RadioBtnEvent {
+    UPDATE_MODEL_VALUE = 'update:modelValue',
+  }
+
+export interface RadioBtnEvents {
+    [RadioBtnEvent.UPDATE_MODEL_VALUE]: RadioBtnProps['modelValue']
+  }
+
+export interface RadioBtnOption {
+    label: string,
+    value: string,
+    description?: string
+  }
+
+export interface RadioBtnProps {
+    options: RadioBtnOption[],
+    modelValue: string,
+    label: string,
+    errorMessage: string,
+    disabled: boolean,
+    isGroupInline: boolean,
+    isTextInline: boolean
+  }
```

Comment:
```suggestion
export interface RadioBtnProps {
    options: RadioBtnOption[],
    modelValue?: string,
    label?: string,
    errorMessage?: string,
    disabled?: boolean,
    isGroupInline?: boolean,
    isTextInline?: boolean
  }
```
I think options the only property that should be required.

Reply:
ok, done